tasks/base.py

```diff
@@ -283,35 +273,31 @@ def run(self, *args, **kwargs):
             metric_context, f"{self.metrics_prefix}.full_runtime", sync=True
         ):
```

Comment:
this timer will insert a SQL row when it's over, but the db commit happens inside the timed code. so the inserted row happens in a separate transaction that is presumably rolled back when the connection drops, or something. so i think if you want to clean things up you can kill this outer timer while you're at it